tests/lib/spread/rules/main.yaml

```diff
@@ -0,0 +1,23 @@
+rules:
+  tests:
+    from:
+      - tests/main/.*
+      - tests/core/.*
+      - tests/completion/.*
+      - tests/cross/.*
+      - tests/regression/.*
+      - tests/smoke/.
+      - tests/unit/.*
+      - tests/upgrade/.*
+    to: [$SELF]
+
+  unit:
+    from: [.*_test.go]
+    to:
+     - tests/unit/go
+     - tests/unit/c-unit-tests-clang
+     - tests/unit/c-unit-tests-gcc
+
+  rest:
```

Comment:
is rest special and only used if the changed file didn't match any other rule?

Reply:
right, it is for the other files which are not matching with previous rules.